mcs/class/Mono.Security/Mono.Security.Interface/MonoTlsProvider.cs

```diff
@@ -163,5 +163,14 @@ internal MonoTlsProvider ()
 			X509CertificateCollection certificates, bool wantsChain, ref X509Chain chain,
 			ref MonoSslPolicyErrors errors, ref int status11);
 #endregion
+
+#region Misc
+
+		internal abstract bool SupportsCleanShutdown {
```

Comment:
Why to add unused property?

Reply:
It is used by the test suite via reflection.  We still need to be able to send the close_notify for testing - and support the scenario where some server sends it to us.  I don't want to make it public yet because that'd mean committing on API compatibility.